app/code/Magento/Sales/Test/Unit/Model/Order/CreditmemoFactoryTest.php

```diff
@@ -49,10 +49,14 @@ class CreditmemoFactoryTest extends TestCase
      */
     protected function setUp(): void
     {
-        $this->orderItemMock = $this->createPartialMock(
-            Item::class,
-            ['getChildrenItems', 'isDummy', 'getHasChildren', 'getId', 'getParentItemId']
-        );
+        $this->orderItemMock = $this->getMockBuilder(Item::class)
+            ->disableOriginalConstructor()
+            ->disableOriginalClone()
+            ->disableArgumentCloning()
+            ->disallowMockingUnknownTypes()
```

Comment:
**YAGNI** If original code is not performing any `clone` then you don't need to provide these methods.